package.json

```diff
@@ -13,16 +13,23 @@
   "keywords": [
     "BuckleScript"
   ],
-  "author": "Cryptonomic Inc",
-  "license": "GPL",
+  "author": "",
+  "license": "MIT",
   "dependencies": {
+    "bs-conseiljs": "git+https://github.com/Cryptonomic/ConseilJS-ReasonML-Bindings.git#9d88c727b96fcebe437ea6b2d3819c1083927b74",
```

Comment:
This is fine for now but w should publish `bs-conseiljs` to NPM soon.

Reply:
can it be part of the regular conseiljs package?

Reply:
No, we have to publish new npm dependency.